tests/bulk/bulkwrite-update-arrayFilters-001.phpt

```diff
@@ -1,15 +1,15 @@
 --TEST--
 MongoDB\Driver\BulkWrite::update with arrayFilters
---XFAIL--
-START() tests must be reimplemented (PHPC-1179)
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
-<?php START('THROWAWAY', ["version" => "36-release"]); CLEANUP(THROWAWAY); ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
+<?php skip_if_not_clean(); ?>
```

Comment:
This is a bit misleading, innit? ``skip_if_not_clean`` will also clean first?

Reply:
This change was made during https://github.com/mongodb/mongo-php-driver/pull/787.

Technically, the function attempts to drop the test collection and skips the test if that fails for some reason. I'll admit it's unlike other `skip_if` functions because it attempts to change the state of the system; however, it can actually skip if something goes wrong. In the past, `CLEAN()` ignored any errors.

If you have some ideas for renaming or revising this, a new ticket would be best :+1: